test/python/circuit/test_control_flow_builders.py

```diff
@@ -37,6 +37,10 @@ class SentinelException(Exception):
     """An exception that we know was raised deliberately."""
 
 
+def _reg_key(reg):
+    return reg.name
```

Comment:
Is this to get around a `lambda` lint rule or something?

Reply:
No, just centralisation because I wrote it a few times.  In reality, I shouldn't have done it at all - when I first wrote the test, I didn't realise that I could put registers in a `set`.

Reply:
Changed to the set they always should have been in 63099d9.